docs/docs/examples/core-components/point-attractor.md

```diff
@@ -0,0 +1,217 @@
+  attractor.
+
+Usually, the goal of a Point Attractor is to move a robot to a desired point in space. The predicates of the component
+help to detect when that is the case such that other events can be triggered;
+
+- Is in linear range: This predicate is true whenever the linear distance between input and attractor is below a
+  configurable threshold (see parameters section below).
+- Is in angular range: This predicate is true whenever the angular distance between input and attractor is below a
+  configurable threshold (see parameters section below).
+- Is in range: This predicate is true whenever the linear and angular range predicates are both true.
+
+<div class="text--center">
+  <img src={pointAttractor} alt="Point Attractor interfaces" />
+</div>
+
+### Parameters
+
+Changing the parameters of the component will tune the response and decide at which threshold the predicates switch;
+
+- Linear and angular gains: As shown
+  [here](../../concepts/robotics-concepts/motion-generation.md#point-attractor-dynamical-system), the function of the DS
+  contains an additional scaling constant $K$, represented by the linear and angular gains parameters here. The higher
+  these values are, the higher the generated velocity. Both parameters are of type vector, so it is also possible to set
+  different gains per axis, or even disabling motion along some axis entirely.
+  :::warning
+  Setting high gains can lead to fast movements and overshoot.
+  :::
+- Maximal linear and angular velocity: Before publishing the output twist, the component clamps the linear and angular
+  velocities to the values of those two parameters. This allows to increase the gains without generating an excessive
+  response, for example.
+- Linear and angular precision threshold: The linear and angular distances from the attractor for the input to qualify
+  as in linear range and in angular range (see predicates above).
+
+## Example
+
+To set up this example, follow these steps:
+
+1. Create a new application and record the last frame of the robot of your choice as described
+   [here](../guides/application-frames.md#record-frame-in-3d-scene-view).
+2. In the same application, add the `Frame to Signal` component. Open its parameter settings, turn on auto-configure and
+   auto-activate and set the required parameter `Frame` to the name of the recorded frame from step 1.
+   :::tip
+   Learn more about the `Frame to Signal` component [here](../../concepts/building-blocks/frames.md#frame-to-signal).
+   :::
+3. Add the `Signal Point Attractor` to the graph and enable auto-configure and auto-activate. Then, connect the Pose
+   output of the component from step 2 with the Attractor pose input. Also connect the Cartesian state output of the
+   Robot State Broadcaster in the hardware with the Input pose input.
+4. Add the `IK Velocity Controller` to the hardware, enable auto-load and auto-activate, and connect the Output twist of
```

Comment:
```suggestion
4. Add the `IK Velocity Controller` to the hardware, enable auto-load and auto-activate, and connect the `Output twist` of
```

another personal preference comment. I just think it's weird to capitalize "Output" but then have "twist" after a space. Bold would work too, I just think it's not as clear this refers to a reserved keyword (same above for other signals). 

At this point we might want to establish a standard way to handle this across docs. We have some comments here and there but not sure we have internal guidelines for this (correct me if I'm wrong)

Reply:
I share that concern yes. I try to use code blocks normally but if I do that here, this is what we end up with and I don't find it visually attractive. Also, I would already have to use code blocks in the interfaces above in a sense.
<img width="949" height="157" alt="image" src="https://github.com/user-attachments/assets/2f6af0bd-98df-4a2b-90cf-521670150e7c" />

I'm not sure what the best approach is, maybe others can weigh in as well. I also struggle with this accross the whole docs

Reply:
The version without blocks was harder?

Reply:
for me yes. I mean any visual queue that this is something reserved would make sense to me, but it's still readable as is. I see how it might not always be very nice, it's just a little more visually verbose imo. 

Again, not strong opinion though. I am also asking for my future PRs